string id # Name of the drone for example drone_1
nav_msgs/Odometry odometry
geometry_msgs/Twist course
move_base_msgs/MoveBaseActionGoal wp
